packages/tiptap/src/shared/extensions/placeholder.tsx

```
@@ -92,12 +92,16 @@ export const Placeholder = Extension.create<PlaceholderOptions>({
                   && placeholderContent !== null
                   && "type" in placeholderContent
                 ) {
+                  const wrappedContent = createElement(PlaceholderErrorBoundary, {
+                    children: placeholderContent as ReactElement,
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use additional arguments to React.createElement

(lint/correctness/noChildrenProp)